newrelic.js

```diff
@@ -0,0 +1,11 @@
+const cfenv = require("cfenv")
+const appEnv = cfenv.getAppEnv()
+const newRelicEnv = appEnv.getServiceCreds("analytics-reporter-new-relic")
```

Comment:
This is tying the implementation pretty tightly to Cloud Foundry. It would be better if we could more easily support e.g. Heroku deployments (I know at least one production user of analytics-reporter in Heroku, the City of Philadelphia) or other deployment environments. 

I hate adding an env var for every single thing -- is it easy to have an app detect whether or not it's in a Cloud Foundry environment? Otherwise, an env var for Cloud Foundry might be the right call here.